### PR TITLE
updated Groovy to Kotlin DSL (including maven-push.gradle)

### PR DESCRIPTION
**Pull request by mani-w3w**
AGP 8.5.0
CompileSDK 34

### CODE REVIEW

File: settings.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven(
+            "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+        )
+        maven(
+            "https://s01.oss.sonatype.org/content/repositories/comwhat3words-1423"
```
Comment (kaytran2992) [inline settings.gradle.kts:17]: Hey @mani-w3w, any mistake on this  "https://s01.oss.sonatype.org/content/repositories/comwhat3words-1423"
?

Comment (mani-w3w) [reply settings.gradle.kts:17]: Ah yeah that was there already, we didn't remove when we released last time, will remove it now, nice one

**Review by QuocNguyen-w3w — APPROVED** [review]
LGTM!

**Review by kaytran2992 — APPROVED** [review]
LGTM

**Review by phpduy — APPROVED** [review]
LGTM!

**Pull request merged**